app/controllers/api/v1/search_controller.rb

```diff
@@ -503,9 +503,25 @@ def set_search_facets_and_filters
             facet = SearchFacet.find_by(identifier: facet_id)
             if facet.present?
               matching_filters = []
-              facet.filters.each do |filter|
-                if filter_values.include?(filter[:id])
-                  matching_filters << filter
+              if facet.is_numeric?
```

Comment:
this method is getting long, extract this part into a helper method?

Reply:
yeah that makes sense